contents/tutorials/performance-metrics.md

```diff
@@ -0,0 +1,141 @@
+---
+title: How to pinpoint the culprits behind your slow web pages
```

Comment:
```suggestion
title: How to improve web app performance using PostHog session replays
```

Reply:
Open to suggestions on this headline

Reply:
"How to improve app performance using session replays" is good imo